c2corg_api/tests/views/test_outing.py

```diff
@@ -362,6 +362,112 @@ def test_post_missing_title(self):
         errors = body.get('errors')
         self.assertEqual(len(errors), 3)
 
+    def test_post_date_start_is_tomorrow(self):
+        tomorrow = (datetime.date.today() + datetime.timedelta(days=1)).strftime('%Y-%m-%d')
+        body_post = {
+            'activities': ['skitouring'],
+            'date_start': tomorrow,
+            'date_end': '2016-01-01',
```

Comment:
It will fail with the start < end test otherwise

```suggestion
            'date_end': '3999-01-01',
```